### ISSUE TITLE
Update the online minifier 

### ISSUE THREAD

**Issue by laurentlb** [laurentlb/Shader_Minifier#145]
- The tool needs a UX update.
- It should be more mobile friendly.
- It should use the latest version of Shader Minifier 
- It would be nice if it supported multiple versions of Shader Minifier.

https://ctrl-alt-test.fr/minifier/

**Comment by laurentlb** [comment]
Current status:
- I improved the interface a bit.
- After an update, everything broke and I'm no longer able to run Shader Minifier on the server.

So it has been broken for almost two months now.

The good news is that I have now a new plan. This evening, I've managed to compile Shader Minifier to Wasm and it can run on a browser. There will be a bit of work needed to plug it in the interface, but it seems promising.
![image](https://user-images.githubusercontent.com/485572/194964839-eea20b31-bed8-4235-8528-71e4600191d8.png)
